frontend/elements/src/entry/home/TOSORT/footer-kidsafe.ts

```diff
@@ -34,14 +44,16 @@ display:flex;
   term:string = ""; 
```

Comment:
these should be hardcoded in the element, they aren't changed at runtime

Reply:
Fixed